omnipaxos/src/omni_paxos.rs

```diff
@@ -210,9 +265,11 @@ where
         self.seq_paxos.append(entry)
     }
 
-    /// Propose a reconfiguration. Returns error if already stopped or new configuration is empty.
-    pub fn reconfigure(&mut self, rc: ReconfigurationRequest) -> Result<(), ProposeErr<T>> {
-        self.seq_paxos.reconfigure(rc)
+    /// Propose a reconfiguration. Returns error if already stopped.
+    pub fn reconfigure(&mut self, new_configuration: ClusterConfig) -> Result<(), ProposeErr<T>> {
+        // TODO: doesn't check to make sure the new_config is valid. A user could propose a flawed
+        // ClusterConfig and if it gets decided be stuck with it. Is this acceptable?
```

Comment:
Perhaps we can extend `ProposeErr` with another variant `InvalidClusterConfig` that returns the ClusterConfig